### PR TITLE
docs: update deployment docs

### PR DESCRIPTION
**Pull request by dl239**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

### CODE REVIEW

File: docs/en/reference/sql/deployment_manage/DEPLOY_STATEMENT.md [hunk 1]
```diff
@@ -22,56 +22,64 @@ DEPLOY deployment_name SELECT clause
...
-CREATE TABLE t1(col0 STRING);
+CREATE TABLE demo_table1(c1 string, c2 int, c3 bigint, c4 float, c5 double, c6 timestamp, c7 date);
 -- SUCCEED: Create successfully
 
-DEPLOY demo_deploy select col0 from t1;
+DEPLOY demo_deploy SELECT c1, c2, sum(c3) OVER w1 AS w1_c3_sum FROM demo_table1 WINDOW w1 AS (PARTITION BY demo_table1.c1 ORDER BY demo_table1.c6 ROWS BETWEEN 2 PRECEDING AND CURRENT ROW);
 -- SUCCEED: deploy successfully
 ```
 
 查看部署详情：
```
Comment (lumianph) [inline docs/en/reference/sql/deployment_manage/DEPLOY_STATEMENT.md:39]: 我们可以使用 `SHOW DEPLOYMENT demo_deploy` 命令查看部署的详情，执行结果如下：

File: docs/zh/reference/sql/deployment_manage/DEPLOY_STATEMENT.md [hunk 2]
```diff
@@ -22,56 +22,64 @@ DEPLOY deployment_name SELECT clause
...
-CREATE TABLE t1(col0 STRING);
+CREATE TABLE demo_table1(c1 string, c2 int, c3 bigint, c4 float, c5 double, c6 timestamp, c7 date);
 -- SUCCEED: Create successfully
 
-DEPLOY demo_deploy select col0 from t1;
+DEPLOY demo_deploy SELECT c1, c2, sum(c3) OVER w1 AS w1_c3_sum FROM demo_table1 WINDOW w1 AS (PARTITION BY demo_table1.c1 ORDER BY demo_table1.c6 ROWS BETWEEN 2 PRECEDING AND CURRENT ROW);
 -- SUCCEED: deploy successfully
 ```
 
 查看部署详情：
```
Comment (lumianph) [inline docs/zh/reference/sql/deployment_manage/DEPLOY_STATEMENT.md]: 我们可以使用 `SHOW DEPLOYMENT demo_deploy` 命令查看部署的详情，执行结果如下：

File: docs/en/reference/sql/deployment_manage/DEPLOY_STATEMENT.md [hunk 3]
```diff
@@ -22,56 +22,64 @@ DEPLOY deployment_name SELECT clause
 
 ### Example: 部署一个SQL到online serving
```
Comment (lumianph) [inline docs/en/reference/sql/deployment_manage/DEPLOY_STATEMENT.md]: this suppose to be an English doc?

**Review by lumianph — CHANGES_REQUESTED** [review]
the English doc has Chinese sentences

**Review by vagetablechicken — APPROVED** [review]

**Review by lumianph — APPROVED** [review]

**Pull request merged**